app/src/main/res/values/strings.xml

```diff
@@ -206,35 +200,35 @@
 
     <!-- This is used to inform the user that there are no entries in the list -->
     <string name="no_entries_found">No Entries Found</string>
-    <!-- This explains the steps the user should take, if no saved entries are found: sign in and sync in Firefox -->
-    <string name="no_entries_description">Firefox Lockbox lets you access passwords you\'ve already saved to Firefox. To
-        view your entries here, you\'ll need to sign in and sync in Firefox.
+    <!-- This explains the steps the user should take, if no saved entries are found: sign in and sync in Firefox. %1$s will be replaced with application name -->
+    <string name="no_entries_description">%1$s lets you access passwords you’ve already saved to Firefox. To
```

Comment:
@sashei `no_entries_description` appears as "%1$s lets you access..." in the no entries state on the entry listing